workspaces/orchestrator/plugins/orchestrator/src/routes.ts

```diff
@@ -15,30 +15,40 @@
  */
 import { createRouteRef, createSubRouteRef } from '@backstage/core-plugin-api';
 
+// orchestrator page
 export const orchestratorRootRouteRef = createRouteRef({
-  id: 'orchestrator',
-});
-
-export const workflowDefinitionsRouteRef = createSubRouteRef({
-  id: 'orchestrator/workflows',
-  parent: orchestratorRootRouteRef,
-  path: '/workflows/:format/:workflowId',
+  id: 'orchestrator', // display WorkflowsTabContent
 });
 
 export const workflowInstancesRouteRef = createSubRouteRef({
   id: 'orchestrator/instances',
   parent: orchestratorRootRouteRef,
-  path: '/instances',
+  path: '/instances', // display WorkflowRunsTabContent
```

Comment:
I got the motivation behind these comments but they can be confusing if not properly maintained in the long term.

I don't think we need to give this hint at all. If you think otherwise, I would suggest to mention the `router.ts` as the reference of mapping. But I think it's obvious in this context.